Nexpo/Controllers/CompaniesController.cs

```diff
@@ -165,6 +165,32 @@ public async Task<ActionResult> PutMe(UpdateCompanySelfDto dto)
             }
             await _companyRepo.Update(company);
 
+            return Ok(company);
+        }
+
+         /// <summary>
+        /// Add new company
+        /// </summary>
+        [HttpPut]
+        [Route("add")]
```

Comment:
This route is not necessary as there is no other POST in companies. The standard route for this controller, i.e "POST api/companies", should be sufficient.